base/libgit2/commit.jl

```diff
@@ -50,8 +50,8 @@ function commit(repo::GitRepo, msg::AbstractString;
                 refname::AbstractString=Consts.HEAD_FILE,
                 author::Signature = Signature(repo),
                 committer::Signature = Signature(repo),
-                tree_id::Oid = Oid(),
-                parent_ids::Vector{Oid}=Oid[])
+                tree_id::AbstractGitHash = GitHash(),
+                parent_ids::Vector{AbstractGitHash}=AbstractGitHash[])
```

Comment:
Before this change, it's always been a vector of concretely-typed hash objects because there was only one hash type. I didn't think letting this be either would cause a problem, since the hashes are passed to `get` individually, and the separate `get` methods return the same type.